pubspec.yaml

```diff
@@ -19,6 +19,11 @@ dev_dependencies:
   build_runner: ^2.0.3
   built_value_generator: ^8.0.0
   dart_flutter_team_lints: ^1.0.0
-  dart_style: ^2.0.0
+  # TODO(srujzs): Restore this to ^2.0.0 once 2.3.4 is released and format the
```

Comment:
I'm not really sure if this is an issue (and it looks like CI isn't happy with it anyways due to the SDK requirement). Should I just keep it as `^2.0.0`?

Reply:
Yeah just keep it at ^2.0.0 (it will still get the latest that it can, on whatever SDK it is running).

We could update the SDK versions of the the job is running the formatter if that is desirable, but the important thing is that your formatter version matches that of the job we are running, otherwise you will have issues.

Reply:
Oh I should have read this comment more fully :).

We should probably increase the minimum to require a version of dart_style that can actually format extension types.

I just chatted with @munificent and it sounds like we can get a release, so we could change this to ^2.3.4.

Reply:
Got it. I guess wait for 2.3.4 then before landing this? I'll do some testing with 2.3.3 and updated SDK constraint to see if any of the bots are unhappy first.

Reply:
as discussed offline either way is OK with me :)